applications/FluidDynamicsApplication/fluid_dynamics_application.h

```diff
@@ -33,12 +33,14 @@
 #include "includes/define.h"
 #include "includes/kratos_application.h"
 
+#include "includes/deprecated_variables.h"
+
 
 // Application includes
 #include "fluid_dynamics_application_variables.h"
 //#include "custom_conditions/fluid_periodic_condition_2d.h"
 #include "custom_elements/vms.h"
-//#include "custom_elements/dynamic_vms.h"
+#include "custom_elements/dynamic_vms.h"
```

Comment:
Again, this file is not using this element (and I should really remove it completely from the repository, but that's another story...).